published/external/afxdp.h

```diff
@@ -7,6 +7,7 @@
 #define AFXDP_H
 
 #include <xdp/hookid.h>
+#include <xdpapi_status.h>
```

Comment:
we mostly don't use underscores in header names, and since this is a minor header file, i think it should be `xdp/status.h`